### PR TITLE
Add eslint for cypress

### PR DESCRIPTION
**Pull request by anxolin**
Add cypress eslint

https://docs.cypress.io/faq/questions/using-cypress-faq.html#Is-there-an-ESLint-plugin-for-Cypress-or-a-list-of-globals

image

image

### CODE REVIEW

**Review by Velenir — APPROVED** [review]
Looks straightforward

Keep in mind, we run linting scripts only on `src` dir, not `cypress` nor `test`, don't know if you want to change that. VSCode extension will of course pick up linting everywhere

**Comment by gnosis-info** [comment]
Travis automatic deployment:
   * **🎩 [Dapp](https://pr1594--dexreact.review.gnosisdev.com)**: Testing web app
   * **📚 [Storybook](https://pr1594--dexreact.review.gnosisdev.com/storybook/)**: Component stories

File: .eslintrc.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ module.exports = {
       },
     },
   ],
-  plugins: ['react-hooks', '@typescript-eslint'],
+  plugins: ['react-hooks', '@typescript-eslint', 'cypress'],
```
Comment (Velenir) [inline .eslintrc.js:39]: What about [plugin:cypress/recommended](https://github.com/cypress-io/eslint-plugin-cypress#recommended-configuration)

Comment (anxolin) [reply .eslintrc.js:39]: u right!

**Comment by Velenir** [comment]
What's the deal with globals?

**Comment by anxolin** [comment]
> What's the deal with globals?

It's related to another PR I'm working on, that waterfalls this one.
Basically, I though I didn't need it at first, then I thought I needed the globals, now I think I don't. Let's see how this ends up :) 

You'll see it.

**Pull request merged**